src/v/cluster/errc.h

```diff
@@ -59,6 +59,7 @@ enum class errc : int16_t {
     error_collecting_health_report,
     leadership_changed,
     feature_disabled,
+    quota_exceeded,
```

Comment:
It's odd to define this in the cluster namespace when the enforcement is all in code in the kafka namespace

Reply:
Old code, nice catch